doc/src/manual/control-flow.md

```diff
@@ -567,17 +574,16 @@ below all interrupt the normal flow of control.
 For example, the [`sqrt()`](@ref) function throws a [`DomainError`](@ref) if applied to a negative
 real value:
 
-```julia
+```jldoctest
 julia> sqrt(-1)
 ERROR: DomainError:
 sqrt will only return a complex result if called with a complex argument. Try sqrt(complex(x)).
- in sqrt(::Int64) at ./math.jl:278
```

Comment:
The point of the example is just to show that it throws a `DomainError`? And if the backtrace is removed like this the doctest doesn't need to be updated every time something change in `math.jl`
Should I include backtraces for all examples then? I have omitted it for some other blocks too.